src/features/courses/course-controller.ts

```diff
@@ -3083,6 +3085,24 @@ class CourseController {
         if (!_.isNil(topic.studentTopicOverride) && topic.studentTopicOverride.length > 0) {
             topic = topic.getWithOverrides(topic.studentTopicOverride[0]) as CourseTopicContent;
         }
+        const unit = await topic.getUnit();
+        const course = await unit.getCourse();
+        const enroll = await course.getEnrolledStudents({
+            where: {
```

Comment:
Missing where clause params that we need
* dropDate is not null
* active true

Reply:
I think you mean dropDate _is_ null

Reply:
Yup, my bad